src/app/core/pages/home/home.component.html

```diff
@@ -114,7 +114,7 @@ <h4>Upcoming activities</h4>
                 <p>Committees</p>
             </div>
             <div class="block col-xs-6 col-md-3">
-                <span class='text-primary h1 info_graphic'>8</span>
+                <span class='text-primary h1 info_graphic'>9</span>
```

Comment:
Can't we make this dynamic? As in check the number of groups in the database of type 'committee'?

Reply:
This is honors tracks. For committees it would be possible, but it does not change so often. There was a new track added so that is why it is now 9